src/styles/main.scss

```diff
@@ -62,6 +62,14 @@ body {
   align-content: center;
   gap: 25px;
 }
+#imageUrl-container {
```

Comment:
The input is very small, not making use of all available space. For example, in a desktop web browser, the URL field is very small, and there is a lot of redundant empty space on either side of the container. 

<img width="915" alt="image" src="https://github.com/jcreek/SelectionWheel/assets/8179928/fba9b660-38cc-425d-96d9-37e02eaabaf3">

On a mobile device it would be great to see the text input on its own line, with the button appearing below it, similar to what happens with the main textarea. 

<img width="330" alt="image" src="https://github.com/jcreek/SelectionWheel/assets/8179928/136f59a0-dcc2-4f8f-993c-3fbc09e22252">

Reply:
This looks great now - nice work!